man/pdfpc.in

```diff
@@ -337,20 +340,21 @@ many minutes you are overtime.
 .SS Notes
 
 .PP
-Textual notes can be displayed for each slide.  A few types of PDF annotations
-are understood by pdfpc and will be automatically imported and displayed (only
-their textual content, no formatting attributes are preserved).  The PDF
-annotations can be made using many PDF editors and even viewers.  These "native"
-PDF notes cannot be edited in pdfpc.
+Textual notes can be displayed for each slide.  pdfpc understands the Markdown
```

Comment:
Maybe I missed something but this PR is independant from the previous Markdown feature? Do you just want to fix the man page in one go?

Reply:
You are right. Strictly speaking, it should have been in the MD patchset. I noticed it when adding man page entry for `--note-format`.